src/System.Globalization.Extensions/tests/Normalization/NormalizationAll.cs

```diff
@@ -17,45 +17,51 @@ private string ConvertToString(string codepoints)
             string[] parts = codepoints.Split('-');
             foreach (string part in parts)
             {
-                sb.Append((char) Int32.Parse(part, NumberStyles.HexNumber | NumberStyles.AllowTrailingWhite | NumberStyles.AllowLeadingWhite, CultureInfo.InvariantCulture));
+                sb.Append((char)Int32.Parse(part, NumberStyles.HexNumber | NumberStyles.AllowTrailingWhite | NumberStyles.AllowLeadingWhite, CultureInfo.InvariantCulture));
             }
             return sb.ToString();
         }
-        
+
         [Fact]
         [ActiveIssue(11803, TestPlatforms.AnyUnix)]
```

Comment:
> [ActiveIssue(11803, TestPlatforms.AnyUnix)] [](start = 7, length = 44)

looks I already fixed this one but some how missed to enable the test. could you please try to remove this one and look if you see any other failures? I'll try to do it too.